duckduckgo/iOS
# privacy protection dashboard

Reviewer: Mia
Asana: https://app.asana.com/0/414235014887631/457416080701390
CC:

**Description**:

Removes old content blocking popover and replaces it with a new privacy protection dashboard.

Note: Zeppelin is not the source of truth yet. Assets used were provided and the site grade in the omnibar as per discussions in Asana.

**Steps to test this PR**:

_Test 1:_
1.  Perform a search and tap on the site grade in the omnibar while it's loading
1. The overview appears and shows the unknown grade with privacy protection on or off as appropriate
1. When the page finishes loading, the site grade is updated

_Test 2:_
1. Go to Evans Cycles and open the privacy protection dashboard
1. The grade should be a B
1. Switch privacy protection off
1. The dashboard shows the "protection off" version of the icons
1. The message "privacy protection disabled" is shown
1. The page reloads in the background and the grade updates once finished
1. The number of trackers text changes from "blocked" to "found"
1. Switch privacy protection on 
1. The dashboard shows the "protection on" version of the icons
1. The page reloads in the background, the grade updates once finished and shows "upgraded from D to B"
1. Add the site to the whitelist and open the dashboard
1. The "privacy protection paused" message is shown

_Test 3:_
1. Go to a page and open the privacy protection dashboard
1. Tap the site grade and the dashboard closes

_Test 4:_
1. Go to a page and open the privacy protection dashboard
1. Enter a search query and submit
1. The privacy protection dashboard closes and shows the page loading

_Test 5:_
1. Go to a page and open the privacy protection dashboard
1. Tap the menu icon
1. The privacy protection dashboard closes and the menu appears

_Test 6:_
1. Go to duckduckgo and open the dashboard
1. The "good privacy practices" text is shown
1. The "encrypted connection" text is shown

_Test 7:_
1. Go to private.brindy.org.uk and open the dashboard
1. The "unknown privacy practices" text is shown
1. The "unencrypted connection" text is shown

_Test 8:_
1. Go to amazon.com and open the dashboard
1. The "some privacy practices" message is displayed

_Test 9:_
1. Visit a range of sites with various grades
1. Open privacy protection dashboard and ensure the right grade is displayed
1. Ensure the number of trackers and major trackers looks correct by comparing with the console


###### Reviewer Checklist:


###### PR DRI Checklist:
